.github/workflows/main.yml

```diff
@@ -29,107 +30,87 @@ jobs:
-        env:
-          INSTALL_LITMUS: true
-          EXPERIMENT_NAME: node-cpu-hog
-          EXPERIMENT_IMAGE: litmuschaos/ansible-runner
-          EXPERIMENT_IMAGE_TAG: ci
-          IMAGE_PULL_POLICY: IfNotPresent
-          LITMUS_CLEANUP: true
+      - name: Installing Cypress and running tests for login page
+        if: startsWith(github.event.comment.body, '/run-e2e-login')
+        uses: cypress-io/github-action@v2
+        continue-on-error: false
+        with:
+          wait-on: http://localhost:3000
+          spec: cypress/integration/Login/*
+          working-directory: litmus-e2e/CypressE2E/
+#           record: true
+#         env:
+#           # pass the Dashboard record key as an environment variable
+#           CYPRESS_RECORD_KEY: ${{ secrets.RECORDING_KEY }}
+
+      - name: Installing Cypress and running tests for Workflow page
+        if: startsWith(github.event.comment.body, '/run-e2e-workflow')
+        uses: cypress-io/github-action@v2
+        continue-on-error: false
+        with:
+          wait-on: http://localhost:3000
+          spec: cypress/integration/Workflow/*
+          working-directory: litmus-e2e/CypressE2E/
+#           record: true
+#         env:
+#           # pass the Dashboard record key as an environment variable
+#           CYPRESS_RECORD_KEY: ${{ secrets.RECORDING_KEY }}
+
+      - name: Install Cypress and run all tests
+        if: startsWith(github.event.comment.body, '/run-e2e-all')
+        uses: cypress-io/github-action@v2
+        continue-on-error: false
+        with:
+          wait-on: http://localhost:3000
+          spec: cypress/integration/**/*
+          working-directory: litmus-e2e/CypressE2E
+#           record: true
+#         env:
+#           # pass the Dashboard record key as an environment variable
+#           CYPRESS_RECORD_KEY: ${{ secrets.RECORDING_KEY }}
 
-      - name: Running node-memory-hog chaos experiment
-        if: startsWith(github.event.comment.body, '/run-e2e-node-memory-hog') || startsWith(github.event.comment.body, '/run-e2e-all')
-        uses: mayadata-io/github-chaos-actions@v0.1.1
-        env:
-          INSTALL_LITMUS: true
-          EXPERIMENT_NAME: node-memory-hog
-          EXPERIMENT_IMAGE: litmuschaos/ansible-runner
-          EXPERIMENT_IMAGE_TAG: ci
-          IMAGE_PULL_POLICY: IfNotPresent
-          LITMUS_CLEANUP: true      
-                      
       - name: Check the test run
         if: |
-         startsWith(github.event.comment.body, '/run-e2e-pod-delete') || startsWith(github.event.comment.body, '/run-e2e-container-kill') ||
-         startsWith(github.event.comment.body, '/run-e2e-node-cpu-hog') || startsWith(github.event.comment.body, '/run-e2e-node-memory-hog') ||
-         startsWith(github.event.comment.body, '/run-e2e-all')
+         startsWith(github.event.comment.body, '/run-e2e-login') || startsWith(github.event.comment.body, '/run-e2e-workflow') || startsWith(github.event.comment.body, '/run-e2e-all')
```

Comment:
same for this, remove if

Reply:
not required here. It's fine.

Reply:
They are required for checking the tests ran or not.